ads/_config.js

```diff
@@ -747,3 +747,14 @@ export const adConfig = {
   },
 
 };
+
+
+/**
+ * Specifies which networks can share master frames.
+ * @type {!Object<string, !Array<string>>}
+ */
+export const masterFrameAccessibleTypes = {
```

Comment:
also could you pls move the settings to individual network.

see reasoning in #5038